app/script/links/LinkPreviewHelpers.js

```diff
@@ -38,7 +38,7 @@ z.links.LinkPreviewHelpers = {
   /**
    * Get first link and link offset for given text.
    * @param {string} text - Text to parse
-   * @returns {Array<string, Number>} First link and its offset
+   * @returns {Array<string, number>} First link and it's offset
```

Comment:
Please correct spelling `@returns {Array<string, number>} First link and its offset` - its is not it is in this case but a genitive thus it`s its